### DISCUSSION TITLE
Encoding issue with accented characters

### DISCUSSION THREAD

**Discussion by ghost** [starship/starship#1801] [Q&A]
Hello, 

it looks like starship doesn't handle accented characters very well.  

With starship

![starship](https://user-images.githubusercontent.com/27742874/96332698-c7842680-1065-11eb-90fc-07ba5f0f8611.png)

Without starship

![powershell](https://user-images.githubusercontent.com/27742874/96332697-c6eb9000-1065-11eb-9a65-fc7dc204f212.png)

Is it something that can be fixed on my part ?

**Comment by davidkna** [comment]
I think this might happen because the powershell starship init changes the console output encoding to `UTF-8` because that's the encoding strings in rust use.

Sadly, it doesn't look like changing the output encoding back within the `prompt` function works. I'll investigate again if there's a way to handle the starship `UTF-8` differently.

**Reply by ghost** [reply]
That's unlucky, I hope you will find something to make it work even if that's not that big of a deal 👍
